test/suites/fips/validate-fips.robot

```diff
@@ -68,6 +74,22 @@ Check Payload Tool Must Pass
     ...    sudo=True    return_rc=True    return_stdout=False    return_stderr=False
     Should Be Equal As Integers    0    ${rc}
 
+Check Container Images In Release Must Pass
+    [Documentation]    Run check-paylod Tool for Release containers
+    ${podman_pull_secret}=    Set Variable    /root/.config/containers/auth.json
+    ${podman_args}=    Set Variable
+    ...    --authfile ${PULL_SECRET_PATH} --rmi --privileged -v ${PULL_SECRET_PATH}:${podman_pull_secret}
```

Comment:
Should we use `--rm` here to clean up the container per-each run?

Reply:
yes, each iteration check-payload is pulling the image into its `/var/lib/containers/storage/` , so we need to remove the `anonymous unnamed volume`  with `-rm`

if we have those images embedded  we can try mounting  them into `/var/lib/containers/storage/`, so it could  save some downloading time for each image..